layersvt/device_simulation.cpp

```diff
@@ -890,6 +1141,14 @@ VKAPI_ATTR VkResult VKAPI_CALL CreateInstance(const VkInstanceCreateInfo *pCreat
                                                   dt->GetPhysicalDeviceQueueFamilyProperties(physical_device, count, results);
                                                   return VK_SUCCESS;
                                               });
+        // Query every standard VkFormat's properties.
```

Comment:
Why pre-populate pdd.array_of_format_properties if GetPhysicalDeviceFormatProperties will call down the chain on lookup failures?

Reply:
The lookup failure that causes it to call down the chain is if the PhysicalDevice has not been overridden.  

If the format lookup fails, an "unsupported" format (all flags==0) is returned, and it does not call down.

Reply:
Aha! I misread that the first time, ignore my other comment